core/tests/snapshots.rs

```diff
@@ -553,7 +553,7 @@ fn test_concurrent_snapshot_packaging(
 
     // files were saved off before we reserialized the bank in the hacked up accounts_hash_verifier stand-in.
     solana_runtime::serde_snapshot::reserialize_bank_with_new_accounts_hash(
-        saved_snapshots_dir.path(),
+        saved_snapshots_dir.path().join(saved_slot.to_string()),
```

Comment:
I noticed in a few places we directly convert slot to string, but in others (https://github.com/solana-labs/solana/pull/31360/files#r1181714591) we use a function: `snapshot_utils::get_snapshot_file_name(slot)`.

Is there a reason for the inconsistency that I'm missing?

Reply:
I can wait till your PR lands first before landing this one.

Reply:
> That fn is `pub(crate)` within the runtime crate, so can't use it here. I already have a PR to fix that once this one lands :)

If you are doing this, could you change the function name to get_bank_snapshot_dir (without s)?  bank_snapshots_dir refers to the the top snapshots directory containing all the snapshot dirs.  bank_snapshot_dir (without s) refers to one specific snapshot dir at a specific slot.  Here the accurate function name should be without s.